src/main/java/games/strategy/triplea/delegate/MustFightBattle.java

```diff
@@ -815,9 +833,24 @@ private void pushFightLoopOnStack(final boolean firstRun) {
 
       @Override
       public void execute(final ExecutionStack stack, final IDelegateBridge bridge) {
+        if( Match.someMatch( m_attackingUnits, Matches.unitHasAttackValueOfAtLeast(1) ) 
```

Comment:
This is a Checkstyle whitespace violation (just run the Eclipse formatter to fix it).